internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -354,6 +354,22 @@ type networkLoadBalancerConfig struct {
 	appDNSName           *string
 }
 
+func convertELBAccessLogsConfig(mft *manifest.Environment) (*template.ELBAccessLogs, error) {
+	elbAccessLogsArgs, isELBAccessLogsSet := mft.ELBAccessLogs()
+	if !isELBAccessLogsSet { //if ELB access logs configs are not defined.
```

Comment:
nit: the code is pretty clear ✨ , I think we can remove these implementation comments